### PR TITLE
DM-36867: fix setFilterLabel

### PR DESCRIPTION
**Pull request by suberlak**

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -1,23 +1,11 @@
 name: lint
 
-# Triggers the workflow on push or pull request events
 on:
-  - push
-  - pull_request
+  push:
+    branches:
+      - main
+  pull_request:
```
Comment (teweitsai) [inline .github/workflows/lint.yaml:7]: Do you need to put any type in `pull_request:`?
https://github.com/marketplace/actions/pull-request-linter

I just felt it is a little weird to me to just see:

```
pull_request:
```

File: bin.src/makeGainImages.py [hunk 2]
```diff
@@ -47,21 +47,21 @@ def main(just_wfs=False, detector_list=None):
     for filt_name in 'ugrizy':
         for ccd in ccd_list:
             name = ccd.getName()
+            print(f'Preparing gain images in filter {filt_name} for {name}')
```
Comment (teweitsai) [inline bin.src/makeGainImages.py]: I thought the `black` will use the double quote?

```
print(f"Preparing gain images in filter {filt_name} for {name}")
```

**Review by teweitsai — APPROVED** [review]
Looks good. Thanks for the quick fix!

File: .github/workflows/lint.yaml (hunk 1 above)
Comment (suberlak) [reply .github/workflows/lint.yaml:7]: I've used the DM-recommended `lint.yaml` https://lsstc.slack.com/archives/C9BEJU1T3/p1667519766208999?thread_ts=1667515906.245289&cid=C9BEJU1T3

Comment (suberlak) [reply .github/workflows/lint.yaml:7]: From https://github.com/lsst/daf_butler/blob/main/.github/workflows/lint.yaml

File: bin.src/makeGainImages.py (hunk 2 above)
Comment (suberlak) [reply bin.src/makeGainImages.py]: Yes, it doesn't seem that black had been run on this file for a while; when I run it it changes that also in all `.set("name")` methods

**Pull request merged**